resources/migrations/000_migrations.yaml

```diff
@@ -1480,15 +1523,7 @@ databaseChangeLog:
   - changeSet:
       id: 10
       author: cammsaul
-      validCheckSum:
-        - 7:3b90e2fe0ac8e617a1f30ef95d39319b
-        - 7:97fec69516d0dfe424ea7365f51bb87e
-        - 8:2e03a495932b4a9aebb9d58a6ad87ca9
-        - 8:431360d062cb82d8b27960b3a0abb98c
-        - 8:5297214d1788d964675d8c6336ac9b6d
-        - 8:532075ff1717d4a16bb9f27c606db46b
-        - 8:96e54d9100db3f9cdcc00eaeccc200a3
-        - 8:9f03a236be31f54e8e5c894fe5fc7f00
+      validCheckSum: ANY
```

Comment:
I got tired of updating these huge lists of checksums all the time, and it's not like we every want to actually fail migrations because we forgot to add a possibility here, so I replaced them all with the magic keyword `ANY` which tells Liquibase to allow any checksum